fluent-plugin-google-cloud.gemspec

```diff
@@ -23,7 +23,7 @@ eos
   gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3'
   gem.add_runtime_dependency 'google-api-client', '> 0.9'
   gem.add_runtime_dependency 'googleauth', '~> 0.4'
-  gem.add_runtime_dependency 'grpc', '= 1.0.1'
+  gem.add_runtime_dependency 'grpc', '~> 1.0', '< 1.0.2'
```

Comment:
On second thought, given your initial comment, it probably makes more sense to use `'< 1.1.0'`, but up to you.